README.md

```diff
@@ -51,6 +51,7 @@ $ gem install cocoapods --pre
 To integrate Alamofire into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
 ```ruby
+use_frameworks!
 source 'https://github.com/CocoaPods/Specs.git'
 platform :ios, '8.0'
```

Comment:
It would be nice if we followed the same structure as the CocoaPods guides. Could you update this section to the following?

```ruby
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '8.0'
use_frameworks!

pod 'Alamofire', '~> 1.1.4'
```